Cargo.toml

```diff
@@ -42,7 +41,18 @@ want = "0.3"
 # Optional
 
 libc = { version = "0.2", optional = true }
+log = { version = "0.4.14", optional = true }
```

Comment:
why is there now a dependency on the `log` crate?

Reply:
> why is there now a dependency on the `log` crate?

Is required to turn off tracing in an app that uses hyper (the example).